src/Indice.Features.Messages.App/src/app/features/campaigns/create/campaign-create.component.ts

```diff
@@ -92,11 +98,12 @@ export class CampaignCreateComponent implements OnInit, AfterViewChecked {
   }
 
   public onStepperStepChanged(event: StepSelectedEvent) {
-    if (event.selectedIndex === 1) {
+    if (event.selectedIndex === 1 && !this.hasTemplateLoaded) {
```

Comment:
simplify if statement. 
1. `event.selectedIndex === 1` I am on the second wizard screen that contains the content views
2. `!this.hasTemplateLoaded === 1` A template id is selected and should be loaded but this has not happened yet.
3.  `if (this.templateId)` The template id is populated so I can make an api call to get the actual contents.

Reply:
The above must be as follows. Three booleans that show clear intent.
1. `isInContentStep`
2. `hasSelectedTemplate`
3. `templateNotLoaded`